README.md

```diff
@@ -1 +1,50 @@
 # Apache Accumulo Testing Suite
+
+The Apache Accumulo testing suite contains applications that test and verify the
+correctness of Accumulo.
+
+## Installation
+
+In order to run the Apache Accumulo testing suite, you will need Java 8 and Maven installed
+on your machine as well as an Accumulo instance to use for testing.
+
+1. First clone this repository.
+
+        git clone git@github.com:apache/accumulo-testing.git
+        cd accumulo-testing
+
+2. All configuation files for the test suite are in `conf/`. Only the `accumulo-testing.props`
+   configuration file needs to be created and edited. All other configuration files are optional.
+
+        cd conf/
+        cp accumulo-testing.props.example accumulo-testing.props
+        vim accumulo-testing.props
+
+3. Tests are run using the `accumulo-testing` command which is located in the `bin/`
+   directory. Run this command without any arguments to view its usage and see available tests.
+
+        ./bin/accumulo-testing
+
+## Random walk test
+
+The random walk test generates client behavior on an Apache Accumulo instance by randomly walking a
+graph of client operations. The test is configured by the `accumulo-testing.props` file and requires
+a test module to be specified when run. See the [modules][modules] directory for available modules.
+The test can be run using single local process or multiple processes on a cluster using YARN.
+
+The command below will start a local random walker using the [Image.xml][image] module.
+
+        ./bin/accumulo-testing rw-local Image.xml
+
+To run multiple random walkers on a cluster using YARN, set `test.container.instances` in
+`accumulo-testing.props` to the number of walkers that you want started and run the command
+below.
+
+        ./bin/accumulo-testing rw-cluster Image.xml
```

Comment:
I'd suggest `rw-yarn` or `rw-twill` instead of `rw-cluster` for some future proofing.